lib/segment/src/index/field_index/full_text_index/mmap_inverted_index/mod.rs

```diff
@@ -1,3 +1,53 @@
 #![allow(dead_code)]
 
+use std::path::PathBuf;
+
+use common::mmap_hashmap::MmapHashMap;
+use mmap_postings::MmapPostings;
+
+use super::inverted_index::ImmutableInvertedIndex;
+use crate::common::mmap_slice_buffered_update_wrapper::MmapSliceBufferedUpdateWrapper;
+use crate::common::operation_error::OperationResult;
+
 mod mmap_postings;
+
+const POSTINGS_FILE: &str = "postings.dat";
+const VOCAB_FILE: &str = "vocab.dat";
+const POINT_TO_TOKENS_COUNT_FILE: &str = "point_to_tokens_count.dat";
+
+pub struct MmapInvertedIndex {
+    path: PathBuf,
+    postings: MmapPostings,
+    vocab: MmapHashMap<str>,
+    point_to_tokens_count: MmapSliceBufferedUpdateWrapper<Option<usize>>,
```

Comment:
Would it be possible to avoid Option? Value = None means deleted? If so, let's use bitmap instead.
`size_of::<Option<usize>>()` = 16 bytes.